apache/incubator-shenyu
# shenyu agent logging #2870

[Volunteer #2870 ] Add log plugin for shenyu-agent #2870 

test method：
1. setup up a rocketmq
2. config shenyu-agent.yaml and set rocketmq props. 
3. package shenyu-agent module.  
4. run org.apache.shenyu.admin.ShenyuAdminBootstrap;
5. add java option  "-javaagent:shenyu-agent.jar path" and run  org.apache.shenyu.bootstrap.ShenyuBootstrapApplication
6. run org.apache.shenyu.examples.http.ShenyuTestHttpApplication;
7.  then send request:
curl --location --request POST 'http://localhost:9195/http/order/save?a=1' \
--header 'Content-Type: application/json' \
--data-raw '{
    "id":666,
    "name": "hello world"
}'
8. then run bellow code:
`    public static void main(String[] args) throws InterruptedException, MQClientException {
        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("shenyu-agent-test-group");
        consumer.setNamesrvAddr("localhost:9876");
        consumer.subscribe("shenyu-agent-logging", "*");
        consumer.registerMessageListener((MessageListenerConcurrently) (msgs, context) -> {
            List<String> msgList = msgs.stream().map(v -> new String(v.getBody())).collect(Collectors.toList());
            for (String msg : msgList) {
                System.out.println(msg);
            }
            return ConsumeConcurrentlyStatus.CONSUME_SUCCESS;
        });
        consumer.start();
        System.out.println("Consumer Started.");
    }`
9.Observe the news of consumer consumption.you can see the result.
